test/integration/dialects/postgres/dao.test.js

```diff
@@ -867,6 +868,35 @@ if (dialect.match(/^postgres/)) {
       });
     });
 
+    it('should save geometry correctly', function() {
+      var point = { type: 'Point', coordinates: [39.807222,-76.984722] };
+      return this.User.create({ username: 'user', email: ['foo@bar.com'], location: point}).then(function(newUser) {
+        expect(point).to.eql(newUser.location);
```

Comment:
Theoretically some internals might mutate `.location`, which we wouldn't notice here. Its better to say `expect({ type: 'Point', coordinates: [39.807222,-76.984722] }).to.deep.eql(newUser.location);`, even though its duplication :)